addons/project_hr_expense/tests/test_project_profitability.py

```diff
@@ -71,3 +71,41 @@ def test_project_profitability(self):
             self.project_profitability_items_empty,
             'No data should be found since the sheet is not approved yet.',
         )
+
+    def test_project_profitability_after_expense_sheet_actions(self):
+        expense = self.env["hr.expense"].create(
+            {
+                "name": "Car Travel Expenses",
+                "employee_id": self.expense_employee.id,
+                "product_id": self.product_a.id,
+                "unit_amount": 50.00,
```

Comment:
```suggestion
                "product_id": self.product_c.id,
                "total_amount": 50.00,
```
Or you'll break your fw-ports for unrelated reasons

Reply:
Thanks. I changed that.